common/scala/src/main/scala/whisk/common/TransactionId.scala

```diff
@@ -39,10 +34,10 @@ import pureconfig._
  */
 case class TransactionId private (meta: TransactionMetadata) extends AnyVal {
   def id = meta.id
-  override def toString = {
-    if (meta.id > 0) s"#tid_${meta.id}"
-    else if (meta.id < 0) s"#sid_${-meta.id}"
-    else "??"
+  override def toString = s"#tid_${meta.id}"
```

Comment:
will a tid now appear as `tid_sid_...` in the logs?

Reply:
Yes, it will.
We thought about removing the `sid_` but we decided to leave it in. If we leave it in, the sids will  attract more attention. Otherwise you see only digits and letters in every tid, and you might miss the sids, as they are also only letters, which might not attract enough attention.

Reply:
heh... not entirely convinced but I don’t think you can easily remove it either so it is fine to let it be. If it becomes annoying we will see it.